ui/job-view/pushes/Push.jsx

```diff
@@ -38,10 +39,19 @@ const fetchTestManifests = async (project, revision) => {
   const rootUrl = prodFirefoxRootUrl;
   const url = `${checkRootUrl(
     rootUrl,
-  )}/api/index/v1/task/gecko.v2.${project}.revision.${revision}.taskgraph.decision/artifacts/public/manifests-by-task.json`;
+  )}/api/index/v1/task/gecko.v2.${project}.revision.${revision}.taskgraph.decision/artifacts/public/manifests-by-task.json.gz`;
   const response = await fetch(url);
   if ([200, 303, 304].indexOf(response.status) > -1) {
```

Comment:
I know this is from before, but you can use `includes` for this.  It's a little cleaner:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes